.github/dependabot.yml

```diff
@@ -6,7 +6,6 @@ updates:
       interval: weekly
       day: tuesday
     reviewers:
-      - "antonlisovenko"
```

Comment:
[q] have you thought about having a group instead, see https://github.com/mongodb/mongocli/blob/aba18a22fa124537e22241a3d56db95358e554e7/.github/dependabot.yml#L9

Reply:
that's a good idea